tests/baselines/reference/esDecorators-classDeclaration-classThisReference(target=es2022).js

```diff
@@ -0,0 +1,45 @@
+//// [esDecorators-classDeclaration-classThisReference.ts]
+declare let dec: any;
+
+@dec
+class C {
+    static { this; }
+    static x: any = this;
+    static accessor a: any = this;
+    static m() { this; }
+    static get g() { return this; }
+}
+
+
+//// [esDecorators-classDeclaration-classThisReference.js]
+let C = (() => {
+    var _a_accessor_storage;
+    let _classDecorators = [dec];
+    let _classDescriptor;
+    let _classExtraInitializers = [];
+    let _classThis;
+    var C = class {
+        static { __setFunctionName(this, "C"); }

```

Comment:
Do we need emitting `setFunctionName` now that `var C = class` should do the trick?